test/metabase/cmd/rotate_encryption_key_test.clj

```diff
@@ -13,11 +13,14 @@
             [metabase.models.interface :as interface]
             [metabase.test :as mt]
             [metabase.test.data.interface :as tx]
+            [metabase.test.fixtures :as fixtures]
             [metabase.util.encryption :as encrypt]
             [metabase.util.encryption-test :as eu]
             [toucan.db :as db]
             [toucan.models :as models]))
 
+(use-fixtures :once (fixtures/initialize :db))
+
```

Comment:
unrelated to this PR but this should fix occasional test failures of `rotate-encryption-key!-test` (hopefully)